### PR TITLE
Only upgrade cartodb sql extension once per database

### PR DESCRIPTION
**Pull request by tylerparsons**

### CODE REVIEW

File: lib/tasks/db_maintenance.rake [hunk 1]
```diff
@@ -236,7 +236,12 @@ namespace :cartodb do
       execute_on_users_with_index(task_name, Proc.new { |user, i|
         begin
           log(sprintf("Trying on %-#{20}s %-#{20}s (%-#{4}s/%-#{4}s)...", user.username, user.database_name, i+1, count), task_name, database_host)
-          user.db_service.load_cartodb_functions(statement_timeout, extension_version)
+          # Only load cartodb extensions once per database, i.e. for dedicated users and org owners
+          load_cartodb_extension = user.account_type == '[DEDICATED]' || user.organization_owner?
```
Comment (vmanohar1) [inline lib/tasks/db_maintenance.rake:240]: Org owner is already dedicated type.. Following should be enough --->
load_cartodb_extension = user.account_type == '[DEDICATED]'

File: app/models/user/db_service.rb [hunk 2]
```diff
@@ -200,7 +202,9 @@ def load_cartodb_functions(statement_timeout = nil, cdb_extension_target_version
           end
         end
 
-        upgrade_cartodb_postgres_extension(statement_timeout, cdb_extension_target_version)
+        if !load_cartodb_extension
```
Comment (vmanohar1) [inline app/models/user/db_service.rb:205]: Don't we want to run it when it is true?

Should it be not - "if load_cartodb_extension" ?

Comment (tylerparsons) [reply app/models/user/db_service.rb:205]: Wrote this initially as `skip_load_cartodb_extension` - must have missed this in the refactor.

File: lib/tasks/db_maintenance.rake (hunk 1 above)
Comment (tylerparsons) [reply lib/tasks/db_maintenance.rake:240]: This is true, but I chose this because it's a bit more readable/failsafe.

**Pull request closed without merging**